cotton/salt/pillar.py

```diff
@@ -41,12 +41,12 @@ def get_unrendered_pillar_locations(include_project=True):
 
 
 def _get_projects_location():
-    fab_location = os.path.dirname(env.real_fabfile)
-    return os.path.abspath(os.path.join(fab_location, '../config/projects/'))
```

Comment:
@tim-mcarthur-moj that line you have commented on is a removal, it has been wrapped in a condition now `if env.use_project_dir:`

Currently this is still there to support older projects, but if we are willing to bump major versions we can deprecate completely